modes/smooth_mode.h

```diff
@@ -5,6 +5,10 @@
 #include "select_cancel_mode.h"
 #include "../common/angle.h"
 
+#if defined(SMOOTH_COLORCHANGE_TICK_SOUND_SENS) && SMOOTH_COLORCHANGE_TICK_SOUND_SENS > 0
+#define SOUND_TICK_ANGLE ((M_PI * 2 / 12) / SMOOTH_COLORCHANGE_TICK_SOUND_SENS)
```

Comment:
Why not just use steps per revolution like everything else?

Reply:
uh.. 
Well, this was ported from working well in 7.14
Also, we're IN smoothmode, so using anything from stepped didn't occur to me.
Not sure how that would work.

Reply:
I just meant to change the unit (and maybe the name) of this define. I did not mean to use anything from stepped mode.

tl;dr; don't divide by 12

Also, the define should not have COLORCHANGE in the name, because this will affect all smooth_mode menus.

Reply:
ok. How about now?

It's set to only execute when in smooth colorchange so as NOT to interfere with other modes.
`    if (SaberBase::GetColorChangeMode() == SaberBase::COLOR_CHANGE_MODE_SMOOTH) {`

Reply:
that is not the right way to do this.
If we only want this to affect color change mode, then the code should go in the color change mode class, not here.

Reply:
> If we only want this to affect color change mode, then the code should go in the color change mode class, not here.

Makes sense.
Moved, tested. Working just as well.
Updated PR files.